app/src/main/java/com/pps/globant/fittracker/mvp/view/LoginView.java

```diff
@@ -0,0 +1,95 @@
+package com.pps.globant.fittracker.mvp.view;
+
+import android.app.Activity;
+
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
+
+import com.google.android.gms.common.SignInButton;
+
+import com.pps.globant.fittracker.R;
+import com.squareup.otto.Bus;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+
+public class LoginView extends ActivityView {
+
+    private Bus bus;
+
+    @BindView(R.id.status)
+    TextView statusLabel;
+    @BindView(R.id.detail)
+    TextView detailLabel;
+    @BindView(R.id.sign_in_button)
+    SignInButton googleSignInButton;
+    @BindView(R.id.sign_out_button)
+    Button googleSignOutButton;
+
+    @BindView(R.id.text_fb_label)
+    TextView textFbLabel;
+
+
+    public LoginView(Activity activity, Bus bus) {
+        super(activity);
+        this.bus = bus;
+        ButterKnife.bind(this, activity);
+    }
+
+    public void setStatusLabel(String label) {
+        statusLabel.setText(label);
+    }
+
+    public void setDetailLabel(String label) {
```

Comment:
Always that you need use the method setText() for a TextView, you could use that manner

    public void setDetailLabel(@StringRes int label) {
        detailLabel.setText(label);
    }

Reply:
done, thanks!